esphome/components/rp2040_pio_led_strip/led_strip.cpp

```diff
@@ -93,6 +108,9 @@ void RP2040PIOLEDStripLightOutput::setup() {
     return;
   }
 
+  // Mark the DMA channel as active
+  dma_chan_active_[this->dma_chan_] = true;
```

Comment:
Can we add some class prefixes here just to make it clearer what is being accessed?
(I know the other static variables are not marked like this, if you want to update them, that would be helpful too, but that would not be a requirement for this PR.)
```suggestion
  RP2040PIOLEDStripLightOutput::dma_chan_active_[this->dma_chan_] = true;
```